src/components/DateTimePicker.js

```diff
@@ -147,85 +149,100 @@ const DatePicker = React.createClass({
     const styles = this.styles();
 
     return (
-      <div style={styles.component}>
-        <div
-          onBlur={this._handleDateBlur}
-          onClick={this._handleDateClick}
-          onFocus={this._handleDateFocus}
-          ref='dateSelect'
-          style={Object.assign({}, styles.selectWrapper, this.state.showCalendar ? styles.activeSelectWrapper : null)}
-          tabIndex={0}
+      <Row style={styles.component}>
```

Comment:
Now that I've updated the grid system, you'll need to require in the new `Container` component and use it to wrap this row.